typescript/src/schema/proto.ts

```diff
@@ -436,6 +442,35 @@ function augmentService(
       '.google.api.oauthScopes'
     ].split(',');
   }
+  augmentedService.pathTemplate = [];
+  for (const property in messages) {
+    if (messages.hasOwnProperty(property)) {
+      const m = messages[property];
+      if (m && m.options) {
+        const option = m.options;
+        if (option && option['.google.api.resource']) {
+          const opt = option['.google.api.resource'];
+          const onePathTemplate = opt as ResourceDescriptor;
```

Comment:
Maybe merge these two lines together? `const resourceDescriptor = option['.google.api.resource'] as ResourceDescriptor;` - there is no need to introduce a new variable.

Reply:
we use this `opt` below to check its type and pattern, not sure we want to combine them into one line, so that would be `option['.google.api.resource']` everywhere